tests/integration/security/util/reachability/context.go

```diff
@@ -135,12 +138,6 @@ func (rc *Context) Run(testCases []TestCase) {
 				return rc.g.DeleteConfig(c.Namespace, policyYAML)
 			})
 
-			// Give some time for the policy propagate.
-			// TODO: query pilot or app to know instead of sleep.
-			ctx.Logf("[%s] [%v] Wait for config propagate to endpoints...", testName, time.Now())
-			time.Sleep(10 * time.Second)
```

Comment:
I'm ok with removing this, but could we have retry longer to compensate on the case query was failed due to configures are not propagated fully. It will be slow when the tests is flow, but should be the same when it works.

I was thinking to add smt to check if policy has been push here before start sending the request, but it could be another improvement.